### DISCUSSION TITLE
File not Found

### DISCUSSION THREAD

**Discussion by cogentax** [nav-gov-hu/M2M#28] [Kérdések] [answered]
Sziasztok,

Az éles és a teszt rendszeren is ua. a hibánk van. Feltöltjük aaó teszt  állomány a tesztre, átfut az ellenőrzésen...és Unknown_File_ID hibát kapunk, ha visszakérdezünk.
Mi lehet a probléma?

Teszt adatok:
Correlation id: 8b95a833-fdad-473b-bec1-62925344cabd
Token request message id: 20c5c249-3cf4-4d43-8a27-4d7d8cac5ddb
File upload message id: 073181e8-8d9d-4aa7-ba08-370dedd6dd29
File id : fe2b9456-b0a4-4b18-a9a4-10b3665b4631
File scan message id: ec383466-5515-4ed2-a369-ed229d5bfb0b
Get document message id: eb4db97c-0f03-432f-bb5b-1fc4408d9b3b
 
{'resultMessage': None, 'documentStatus': None, 'documentFileId': 'fe2b9456-b0a4-4b18-a9a4-10b3665b4631', 'errors': None, 'arrivalNumber': None, 'resultCode': 'UNKNOWN_FILE_ID'}

**Comment by TamasPill** [comment]
A fájl státusz lekérdezéssel volt a probléma, ami kihatott a bizonylat létrehozásra is.
A hiba maga elhárult, de a forrását még keressük.
Addig a createDocument-től lehet folytatni a folyamatot.

**Reply by cogentax** [reply]
Sziasztok,

Egy kapcsolódó kérdés.
Van maximum ideje a vírusellenőrzésnek, és a validálásnak?

Köazi!

**Reply by TamasPill** [reply, marked as answer]
Maximum ideje nincsen, viszont átlagos válaszidők publikálásával készülünk.
